source/UF/Pushouts.lagda

```diff
@@ -0,0 +1,418 @@
+Ian Ray, 15th January 2025
+
+Edited by Ian Ray on 16th March 2025.
+
+Pushouts are defined as a higher inductive type (in the form of a record type). We
+postulate point and path constructors and the (dependent) universal property.
```

Comment:
Perhaps the word "postulate" here is not good. We don't use postulates in TypeTopology. Maybe write "assume".